ccdImports/configFiles/CCD_Probate_Backoffice/CaseEvent.json

```diff
@@ -136,5 +136,5 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "SYSTEM_MAINTENANCE", "Name": "System maintenance", "Description": "Maintenance updates to cases, made automatically by the system", "DisplayOrder": 126, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public", "ShowSummary": "N"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation","ID": "assembleLetter","Name": "Assemble a letter","Description": "Assemble a letter","DisplayOrder": 126, "PreConditionState(s)": "BOCaseStopped","PostConditionState": "BOCaseStopped","CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/document/generateLetter","SecurityClassification": "Public","ShowSummary": "N"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation","ID": "assembleLetterForReissue","Name": "Assemble a letter","Description": "Assemble a letter","DisplayOrder": 127, "PreConditionState(s)": "BOCaseStoppedReissue","PostConditionState": "BOCaseStoppedReissue","CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/document/generateLetter","SecurityClassification": "Public","ShowSummary": "N"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "createCaseFromBulkScan", "Name": "Case created from bulk scan", "Description": "Create a case from an application", "DisplayOrder": 128, "PostConditionState": "CaseCreated", "SecurityClassification": "Public", "ShowSummary": "N"}
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "createCaseFromBulkScan", "Name": "Case created from bulk scan", "Description": "Create a case from an application", "DisplayOrder": 128, "PostConditionState": "CaseCreated", "CallBackURLAboutToSubmitEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/notify/grant-received", "RetriesTimeoutURLAboutToSubmitEvent": "60", "SecurityClassification": "Public", "ShowSummary": "N"}
```

Comment:
What was the reason behind the 60second retry?

Reply:
I was worried the service would timeout and not send an email/letter. With the 60 seconds, if a timeout occurs the process will kick off again which is ok as it just attempts to send a notification email/letter, it's not changing the case in any way.